README.rst

```diff
@@ -391,10 +392,11 @@ Lightning-Qubit should be installed before Lightning-Tensor (compilation is not
     git clone https://github.com/PennyLaneAI/pennylane-lightning.git
     cd pennylane-lightning
     pip install -r requirements.txt
+    pip install cutensornet-cu12
```

Comment:
I think for LTensor we also need to add `cutensornet-cu12`. What do you think? :thinking: 
```suggestion
    pip install cutensornet-cu12 cutensornet-cu12
```

Reply:
thanks @LuisAlfredoNu ! Do we want to install `cutensornet-cu12` twice?

Reply:
hahah sorry, I copied the wrong line. :sweat_smile:  I was referring to `custatevec-cu12`. What do you think?
Also, it might not be for this PR but we should add a library checking in LTensor for `custatevec-cu12` and `cutensornet-cu12`

Reply:
I thought `custatevec-cu12` is not required for LTensor. Could you please let me know the scenarios when `custatevec-cu12` is required? Thanks @LuisAlfredoNu